cfgmgr/buffermgr.cpp

```diff
@@ -34,6 +34,7 @@ void BufferMgr::readPgProfileLookupFile(string file)
     ifstream infile(file);
     if (!infile.is_open())
     {
+        m_pgfile_exist = false;
         return;
     }
```

Comment:
assign m_pgfile_exist true here, and you could remove class member initialization in header file.

Reply:
Either way should do the work, any reason you prefer to put here rather than in the header file?

Reply:
For better readability.

Reply:
I don't see any confusion with the initialization in header file.  it is easily readable so I kept the code here.